package/modem-manager/modem-manager.mk

```diff
@@ -41,6 +41,10 @@ else
 MODEM_MANAGER_CONF_OPTS += --disable-introspection
 endif
 
+ifeq ($(BR2_PACKAGE_MODEM_MANAGER_AT_INTERFACE),y)
+MODEM_MANAGER_CONF_OPTS += --with-at-command-via-dbus
```

Comment:
no without? on the else?

Reply:
Doesn't exist.  The default is to omit this flag which renders the interface inaccessible outside of debug mode.